components/back-button/back-button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./back-button.css";
+import SVG from "react-inlinesvg";
+import leftArrow from "../../public/chevron-left.svg";
```

Comment:
you can import everything from public folder directly inside the public folder.
this is an example:

//@ts-ignore
import signoutIcon from '/images/svgs/sign-out-alt.svg';

the folder images is inside public. also is importance to notice the //@ts-ignore.